arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1142 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Log Analytics Client for LinkTargets, StorageInsightConfigs and SavedSearches.",
+    "description": ".Net client wrapper for the REST API for Azure Log Analytics",
+    "version": "2015-03-20"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
```

Comment:
Does the service actually support text/json? We actually tried this in storage management and found that the service returns a 409 and says that "text/json" is not supported. Can you test the same via Fiddler and make sure that this is correct ?